templates/components/cart/content.html

```diff
@@ -23,7 +23,7 @@
                             image=image
                             class="cart-item-image"
                             fallback_size=../theme_settings.productthumb_size
-                            lazyload='disabled'
```

Comment:
Hi @bookernath! Need your review. We use disabled value for lazyload and because of that lazyload class is not sertting to responsive-img component. Because of that lazysizes library ignores images from cart list. This lib is responsive to set size attribute to the image to help browser decide which url he should use to load the images. As we have no sizes here, browser chose url from srcset due to viewport (screen size in this case) and loading to big images. Because of loading full size images we have delay on its loading. Any major reasons to use disabled here or we can just use ../theme_settings.lazyload_mode, as we using it for such type of images in similar entities (cart preview list and account order list)?

Reply:
My reason for disabling here was mostly because the Cart page does not have much other content in terms of images, and the cart images are above-the-fold high priority content. So this was a small optimization for user experience, to avoid delaying the loading of these key images.

However, I am fine with this change if it fixes a problem. 👍